.github/workflows/misc-migtests.yml

```diff
@@ -92,12 +92,12 @@ jobs:
           AWS_SECRET_ACCESS_KEY: ${{ secrets.RAHULB_S3_SECRET_ACCESS_KEY }}
         if: ${{ !cancelled() }}
         run: |
-          versions=("2024.2.0.0-b145" "2.20.8.0-b53" "2024.1.3.1-b8" "2.23.1.0-b220")
+          versions=("2024.2.1.0-b185" "2.20.9.0-b47" "2024.1.4.0-b108" "2.23.1.0-b220")
           for version in "${versions[@]}"; do
             echo "Running tests on version $version"
 
             echo "Start YugabyteDB cluster"
-            docker run -d --name yugabytedb-$version \
+            docker run -d --name yugabytedb-$version --hostname yb-cluster-test-$version\
```

Comment:
in etc/hosts, we're adding `yb-cluster-test`, but here we are specifying `yb-cluster-test-$version`. Don't understand this discrepancy...

Reply:
Hmm, yeah it was a miss and there is no failure in this workflow because the yb_servers host is not used as such in these tests
Fixed it now